src/applications/simple-forms/21-0845/containers/IntroductionPage.jsx

```diff
@@ -58,46 +64,89 @@ class IntroductionPage extends React.Component {
         <p>
           <a href="https://ask.va.gov/">Contact us online through Ask VA</a>
         </p>
-        {/* TODO: Use <va-telephone> once DST's fixed the 1-800 display bug: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1932 */}
-        {/* <va-telephone contact="18008271000" /> */}
         <p>
-          Or call us at{' '}
-          <a href="tel:+18008271000" aria-label="1. 8 0 0. 8 2 7. 1 0 0 0.">
-            1-800-827-1000
-          </a>{' '}
-          (
+          Or call us at <va-telephone contact="8008271000" /> (
           <va-telephone tty="true" contact="711">
             TTY:711
           </va-telephone>
           ). We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.
         </p>
-        <SaveInProgressIntro
-          formConfig={formConfig}
-          headingLevel={2}
-          alertTitle="Save time and save your work in progress by signing in"
-          prefillEnabled={formConfig.prefillEnabled}
-          messages={formConfig.savedFormMessages}
-          pageList={pageList}
-          startText="Start your authorization"
-          unauthStartText="Sign in to start your authorization"
-          verifiedPrefillAlert={
-            <div>
-              <div className="usa-alert usa-alert-info schemaform-sip-alert">
-                <div className="usa-alert-body">
-                  <strong>Note:</strong> Since you’re signed in to your account,
-                  you can save your release authorization in progress and come
-                  back later to finish filling it out.
+        <h2 id="start-your-request">Start your authorization</h2>
+        <p>
+          <strong>Note</strong>: You’ll need to sign in with a verified{' '}
+          <strong>Login.gov</strong> or <strong>ID.me</strong> account or a
+          Premium <strong>DS Logon</strong> or <strong>My HealtheVet</strong>{' '}
+          account. If you don’t have any of those accounts, you can create a
+          free <strong>Login.gov</strong> or <strong>ID.me</strong> account when
+          you sign in to start filling out your form.
+        </p>
+        {userLoggedIn &&
+        !userIdVerified /* If User's signed-in but not identity-verified [not LOA3] */ && (
+            <div className="id-not-verified-content vads-u-margin-top--4">
+              <va-alert status="continue" uswds>
+                <h3 slot="headline">
+                  You’ll need to verify your identity to authorize the release
```

Comment:
Are we going to start adding this to every form? if so, that makes a nice case for updating this in yeoman and doing that work

Reply:
If 3 out of the 10 last forms we digitized is any indication, I'd say Yes, @rhasselle-oddball .  We could definitely use this as a reusable feature.

Reply:
^Actually, 4 out of 10 forms now.  I just got another LOA3 tic.